docs/troubleshooting.rst

```diff
@@ -162,3 +162,34 @@ memory.
 To fix this, increase the open file limit for non-root user::
 
   $ ulimit -n 1024
+
+
+Authentication failed. authType is ... and userID is ...
+========================================================
+
+This error can be seen through the kinto-admin, but it arises from a limitation
+present in the multiauth.policy configuration.
+
+You must make sure that two authorization policies won't match a given token,
+as this will cause the incoherent userID behaviors.
+
+This was specifically noticed while using Auth0 and Google as multiauth policies.
+Here is an example of one such conflicting configuration::
+
+   multiauth.policies = auth0 google
+
+   multiauth.policy.google.use = kinto.plugins.openid.OpenIDConnectPolicy
+   multiauth.policy.google.issuer = https://accounts.google.com
+   multiauth.policy.google.client_id = <google client id>
+   multiauth.policy.google.client_secret = <google client secret>
+   multiauth.policy.google.userid_field = email
+
+   multiauth.policy.auth0.use = kinto.plugins.openid.OpenIDConnectPolicy
+   multiauth.policy.auth0.issuer = https://minimal-demo-iam.auth0.com
+   multiauth.policy.auth0.client_id = <auth0 client id>
+   multiauth.policy.auth0.client_secret = <auth0 client secret>
+   multiauth.policy.auth0.userid_field = email
+
+One simple solution is not to use Auth0 in conjunction with other auth policies.
```

Comment:
*with other auth policies that overlap with the ones it supports.*

...or something like that.  Because it is perfectly fine to use Auth0 with accounts etc.

Reply:
True! Had not thought about that. I'll change that and add a link to the Auth0 supported Identity providers.